src/test/resources/liquibase/harness/compatibility/foundational/expectedResultSet/redshift/8/createTable.json

```diff
@@ -2,63 +2,83 @@
   "createTable": [
     {
       "comments": "",
+      "dateexecuted": ".*",
       "author": "liquibase",
+      "orderexecuted": 1,
       "description": "sql",
       "contexts": "",
       "exectype": "EXECUTED",
+      "liquibase": "4.31.1",
```

Comment:
this will break tests after next release, we don't check liquibase version in out IT tests